src/AdoNetCore.AseClient/AseDataReader.cs

```diff
@@ -205,7 +201,7 @@ public override long GetChars(int i, long fieldOffset, char[] buffer, int buffer
                 ++cIndex;
             }
 #else
-            Array.Copy(charArray, fieldOffset, buffer, bufferoffset, charsToRead);
+            Array.Copy(charArray, fieldOffset, buffer, bufferOffset, charsToRead);
```

Comment:
Yeah I left the ones that weren't `byte[]` alone because you have to do a bit of math or else `Buffer.BlockCopy` corrupts things.

I think you would do `Buffer.BlockCopy(charArray, fieldOffset, buffer, bufferOffset, charsToRead * sizeof(char));` but I didn't want to break random things.

Reply:
Ah yes, they're chars not bytes